test/paper-item.html

```diff
@@ -91,6 +99,37 @@
           }, 1);
         });
 
+        test('enter triggers a click event', function(done) {
+          MockInteractions.pressEnter(item);
+          Polymer.Base.async(function(){
+            // You need two ticks, one for the MockInteractions event, and one
+            // for the button event.
+            Polymer.Base.async(function(){
+              expect(clickHandler.callCount).to.be.equal(1);
+              done();
+            }, 1);
+          }, 1);
+        });
+
+        test('click triggers a click event', function(done) {
+          MockInteractions.tap(item);
+          Polymer.Base.async(function(){
+            expect(clickHandler.callCount).to.be.equal(1);
+            done();
+          }, 1);
+        });
+      });
+
+      suite('button basic', function() {
+        var item, clickHandler;
+
+        setup(function() {
+          item = fixture('button').querySelector('button');
+          clickHandler = sinon.spy();
+          item.addEventListener('click', clickHandler);
+        });
+
+        // Button also natively fires click after pressing space and enter.
```

Comment:
`MockInteractions` doesn't fire `click` on pressing space/enter (which the native button internally does), so testing that `enter triggers a click event` like we do in `paper-item` would literally involve us calling `click`, which I feel defeats the purpose of the test. However, I am trusting that native buttons get activated on space/enter, which I think is ok.